parachain/pallets/inbound-queue/fixtures/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
We need our license header here. I think perhaps in the generated fixtures too.

Reply:
https://github.com/Snowfork/snowbridge/pull/1125/commits/1da51e135ae0b278568f8b8bbc4232378ec1a4e4